premake5-clean.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+call "premake5.exe" clean
+ping -n 2 127.0.0.1>nul
```

Comment:
What's with the loopback pings here?

Reply:
This line of code is optional. If you prefer, you can ‘skip this line’ or use ‘pause’ instead. I’m too lazy to manually close the command line because the batch script executes too quickly to see the output. It’s similar to sleep(2), and the main purpose of the line is to make the script wait for 2 seconds to see the output before closing.